frontend/src/scenes/session-recordings/settings/SessionRecordingSettings.tsx

```diff
@@ -102,6 +106,192 @@ export function SessionRecordingSettings({ inModal = false }: SessionRecordingSe
+                                {
+                                    label: '5%',
+                                    value: '0.05',
+                                },
+                                {
+                                    label: '0% (replay disabled)',
+                                    value: '0.00',
+                                },
+                            ]}
+                            value={
+                                typeof currentTeam?.session_recording_sample_rate === 'string'
+                                    ? currentTeam?.session_recording_sample_rate
+                                    : '1.00'
+                            }
+                        />
+                    </div>
+                    <p>
+                        Use this setting to restrict the percentage of sessions that will be recorded. This is useful if
+                        you want to reduce the amount of data you collect. 100% means all sessions will be collected.
+                        50% means roughly half of sessions will be collected.
+                    </p>
+                    <div className={'flex flex-row justify-between'}>
+                        <LemonLabel className="text-base">Minimum session duration (seconds)</LemonLabel>
+                        <LemonSelect
+                            dropdownMatchSelectWidth={false}
+                            onChange={(v) => {
+                                updateCurrentTeam({ session_recording_minimum_duration_milliseconds: v })
+                            }}
+                            options={[
+                                {
+                                    label: 'no minimum',
+                                    value: null,
+                                },
+                                {
+                                    label: '1',
+                                    value: 1000,
+                                },
+                                {
+                                    label: '2',
+                                    value: 2000,
+                                },
+                                {
+                                    label: '5',
+                                    value: 5000,
+                                },
+                                {
+                                    label: '10',
+                                    value: 10000,
+                                },
+                                {
+                                    label: '15',
+                                    value: 15000,
+                                },
+                            ]}
+                            value={currentTeam?.session_recording_minimum_duration_milliseconds}
+                        />
+                    </div>
+                    <p>
+                        Setting a minimum session duration will ensure that only sessions that last longer than that
```

Comment:
I think we should warn that this does mean some useful data may not be collected. Debugging a user issue that starts with a loading screen before redirecting to another domain. With this setting set to say, 10 seconds, the initial redirect screen would not have been recorded as it is only buffered in memory.

Reply:
This could also be expanded in documentation instead of the modal